### PR TITLE
DOCS-2912 Write SCALE Data Protection / "Scrub Tasks" article

### PR DESCRIPTION
**Pull request by tonyriv3**
Add a new article to the Data Protection topic: https://www.truenas.com/docs/scale/dataprotection/ . This needs to introduce Disk Scrubbing, explain the default creation of scrub tasks, and document the process and individual fields in a new Scrub Task.

### CODE REVIEW

**Review by BonnieF — CHANGES_REQUESTED** [review]
The area {{< expand "Advanced Scheduler" "v" >}}
{{< include file="static/includes/CORE/AdvancedScheduler.md.part" markdown="true" >}}
{{< /expand >}}

is repeated in the text below the expand box.

**Comment by Mrt134** [comment]
Build test hit false positives for U7 security links that will be live soon. Approved for merge.

File: content/SCALE/DataProtection/ScrubTasksSCALE.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+TrueNAS needs at least one data [pool]({{< relref "/content/SCALE/Storage/Pools/_index.md" >}}) to create scrub task.
+
+To create a scrub task for a pool, go to **Data Protection** and click **ADD** in the **Scrub Tasks** window.
+
+![ScrubTaskNewSCALE](/images/SCALE/ScrubTaskNewSCALE.png "New Scrub Task")
+
+{{< include file="static/includes/Reference/TasksScrubTasksAddFields.md.part" markdown="true" >}}
+
+{{< expand "Advanced Scheduler" "v" >}}
```
Comment (Mrt134) [inline content/SCALE/DataProtection/ScrubTasksSCALE.md:27]: I think this advanced scheduler text is a good candidate to turn into a single source snippet and called in multiple places with an include shortcode. The custom scheduler can be opened when configuring Scrub, Cloud Sync, Periodic Snapshot, Rsync, Replication, and SMART tasks (and possibly other locations in the SCALE UI).

**Review by Mrt134 — COMMENTED** [review]
Great start - I've got one request to single-source the advanced scheduler text (used in several places in the SCALE UI), but the content looks good.

File: content/SCALE/DataProtection/ScrubTasksSCALE.md (hunk 1 above)
Comment (tonyriv3) [reply content/SCALE/DataProtection/ScrubTasksSCALE.md:27]: Fixed!

**Review by Mrt134 — APPROVED** [review]

**Review by BonnieF — APPROVED** [review]

**Pull request merged**